src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+const listItems = [...listEl.children];
+
+const convertToNumber = str => {
+  if (typeof str === 'string') {
+    const cleanedStr = str.replace(/[^\d.-]/g, '');
+    return Number(cleanedStr);
+  }
+  return NaN;
+};
+
+const sortList = (list, sortParam) => {
+  return [...list].sort((a, b) =>
+    convertToNumber(b.dataset[sortParam]) - convertToNumber(a.dataset[sortParam])
+  );
+};
+
+const getEmployees = (sortedList) => {
+  listEl.innerHTML = '';
+
+  sortedList.forEach(item => {
+    listEl.appendChild(item);
+  });
+};
```

Comment:
The name 'getEmployees' suggests that this function retrieves employees from somewhere, but it actually renders a list of employees. Consider renaming this function to better reflect what it does, for example, 'renderEmployees'.